src/Employer/Employer.Web/Orchestrators/Part1/LocationOrchestrator.cs

```diff
@@ -152,41 +179,86 @@ private async Task<LocationViewModel> GetViewModelAsync(Vacancy vacancy, bool? h
             var vm = new LocationViewModel();
             vm.PageInfo = Utility.GetPartOnePageInfo(vacancy);
 
-            vm.LegalEntityLocation = legalEntity.Address.ToString();
-
+            vm.LocationName = legalEntity.Address.ToString();
+            vm.OtherLocationsAddress = employerProfile?.OtherLocations;
+            
             if (vacancy.EmployerLocation != null && (!hasLegalEntityChanged.HasValue || hasLegalEntityChanged == false))
             {
-                vm.UseOtherLocation = false;
 
-                if (vacancy.EmployerLocation.ToString() != legalEntity.Address.ToString())
+                if (vacancy.EmployerLocation.ToString() == legalEntity.Address.ToString())
+                    vm.Location = LegalEntityLocation;
+                else
                 {
-                    vm.UseOtherLocation = true;
-                    vm.AddressLine1 = vacancy.EmployerLocation.AddressLine1;
-                    vm.AddressLine2 = vacancy.EmployerLocation.AddressLine2;
-                    vm.AddressLine3 = vacancy.EmployerLocation.AddressLine3;
-                    vm.AddressLine4 = vacancy.EmployerLocation.AddressLine4;
-                    vm.Postcode = vacancy.EmployerLocation.Postcode;
+                    var employerLocation = vacancy.EmployerLocation.ToString();
+                    StringComparer comparer = StringComparer.OrdinalIgnoreCase;
+                    var otherLocations = vm.OtherLocationsAddress;
+                    if (otherLocations?.Any() == true)
```

Comment:
We can kill this check if setting an empty list above. We don't need otherLocations  variable.